src/sass/_banner.scss

```diff
@@ -0,0 +1,30 @@
+// custom rules applied to gravity ui scss
+header[role=banner] {
+
+  .medium-logo {
```

Comment:
Personally, I'd prefer if we just use our own colours here. I think the Medium logo visually stands out a bit too much as it stands. Besides, we already do this for other social media links in the footer.

You could achieve that by simply setting `fill: currentColor;` - which will make it match whatever the text colour is. That'll automatically work for all themes and states too :-)

Better yet, rather than introducing a new `medium-logo` class - you could simply use our existing `grav-c-icon` and effectively get those CSS behaviours "for free".

Reply:
I'd add that even in that case, I'd suggest to move these static values into scoped variables at the beginning of the class.